public/js/calendarPage.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
dont need this, the file is deferred anyway so will only execute after the content as it is